app/templates/route-not-found.hbs

```diff
@@ -0,0 +1,12 @@
+{{page-title "404 - Pagina niet gevonden"}}
```

Comment:
Is the `404` needed in the title? Seems like a technical detail that the user doesn't really need to know?